tests/utils_tests/test_permutation.py

```diff
@@ -62,5 +62,35 @@ def test_permute_adj(adj):
                 adj[..., perm[i], perm[j]], adj_perm[..., i, j])
 
 
+@pytest.mark.parametrize('adj', [
+    numpy.random.randint(10, size=(batchsize, num_node, num_node, ch),
+                         dtype=numpy.int32)
+])
```

Comment:
It seems `adj` has only one parameter. So I think we do not have to parameterize this test.